sources/packages/web/src/components/generic/DesignationStatusBadge.vue

```diff
@@ -0,0 +1,25 @@
+<!--
+Used when we need to display title and the value inside a container
```

Comment:
Please review this comment. I believe that it came from the other component TitleValue.vue.